test/test_helper.exs

```diff
@@ -1 +1,70 @@
 ExUnit.start()
+
+
+defmodule ThriftTestHelpers do
+  @root_dir "test/fixtures/thrift/"
+
+  defmacro __using__(_) do
+    quote do
+      require ThriftTestHelpers
+      import ThriftTestHelpers
+    end
+  end
+
+  def build_thrift_file(base_dir, {file_name, contents}) do
+    file_relative_path = file_name
+    |> Atom.to_string
+
+
+    file_path = Path.join(base_dir, file_relative_path)
+
+    file_path
+    |> Path.dirname
+    |> File.mkdir_p!
+
+    File.write!(file_path, contents)
+    file_relative_path
+  end
+
+  def tmp_dir do
```

Comment:
I see. The thing that I don't like about this particular approach is that it relies on `mktemp(1)`, which limits the portability of the tests a small amount. It would be better to do this with built-in functions.

Perhaps something more like:

```elixir
tmp_path = Path.join(System.tmp_dir!(), Integer.to_string(System.unique_integer))
System.mkdir tmp_path
```

... or something like that.